torch/autograd/graph.py

```diff
@@ -203,6 +204,21 @@ def increment_version(tensor):
     torch._C._increment_version(tensor)
 
 
+def forbid_in_autograd(tensor):
```

Comment:
forbid_in_backward() ?
It's not forbidden in forward mode AD or from interracting in a non-differentiable way with autograd.